index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Ajax Reddit Slideshow</title>
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+	<div id='container'>
+		<div id='slideshow'>
+			<h1>Slideshow</h1>
+		</div>
+		<div id='search-container'>
+			<input type='text' value='Please enter an item to search for' id='input'>
```

Comment:
Instead of setting a value for this input, let's use placeholder here. That way the default text won't clear when you reset your form.
```
<input type='text' placeholder='Please enter an item to search for' id='input'>
```